src/librustc_resolve/resolve_imports.rs

```diff
@@ -65,7 +65,7 @@ pub struct ImportDirective<'a> {
     pub id: NodeId,
     parent: Module<'a>,
     module_path: Vec<Name>,
-    target_module: Cell<Option<Module<'a>>>, // the resolution of `module_path`
+    module: Cell<Option<Module<'a>>>, // the resolution of `module_path`
```

Comment:
I'm not sure this is an improvement - it seems to introduce ambiguity since there is both the imported/target module and the importing module.